luatest/cluster.lua

```diff
@@ -13,25 +13,24 @@
 --     server:exec(<...>)
 -- end)
 --
--- After setting up a cluster object the following methods could
--- be used to interact with it:
+-- -- After setting up a cluster object the following methods
+-- -- could be used to interact with it:
+-- -- * :start() Startup the cluster.
+-- -- * :start_instance() Startup a specific instance.
+-- -- * :stop() Stop the cluster.
+-- -- * :each() Execute a function on each instance.
+-- -- * :size() get an amount of instances
+-- -- * :drop() Drop the cluster.
+-- -- * :sync() Sync the configuration and collect a new set of
+-- -- instances
+-- -- * :reload() Reload the configuration.
 --
--- * :start() Startup the cluster.
--- * :start_instance() Startup a specific instance.
--- * :stop() Stop the cluster.
--- * :each() Execute a function on each instance.
--- * :size() get an amount of instances
--- * :drop() Drop the cluster.
--- * :sync() Sync the configuration and collect a new set of
---   instances
--- * :reload() Reload the configuration.
+-- -- The module can also be used for testing failure startup
+-- -- cases:
 --
--- The module can also be used for testing failure startup
--- cases:
+-- Cluster.startup_error(config, "something went wrong")
```

Comment:
Can't it be just a prose text? IOW, can we break the `@usage` section or move it upward?

Reply:
Or eliminate usage of the `@usage` section and just mark code blocks my myself?